README.md

```diff
@@ -13,7 +13,7 @@ To upgrade to the latest RubyGems, run:
   $ gem update --system  # you might need to be an administrator or root
 ```
 
-See UPGRADING.rdoc for more details and alternative instructions.
+See [UPGRADING.rdoc] for more details and alternative instructions.
```

Comment:
Actually you are partially right. It should be [UPGRADING](Upgrading.rdoc). I just fixed that. See link below for more details: https://help.github.com/articles/relative-links-in-readmes/